mbuild/compound.py

```diff
@@ -2365,7 +2366,7 @@ def to_networkx(self, names_only=False):
         nodes = list()
         edges = list()
         if names_only:
-            nodes.append(self.name)
+            nodes.append(self.name + '_' + str(id(self)))
```

Comment:
~This will fail if the object has the same name and memory location~ Never mind,  I forgot how `id` worked https://docs.python.org/3/library/functions.html#id